src/pages/AdminCommunication.tsx

```diff
@@ -0,0 +1,20 @@
+import { Box, Toolbar, Typography } from "@mui/material";
+import React from "react";
+import AdminTaskbar from "../components/AdminTaskbar";
+
+const AdminCommunication = () => {
+	return (
+		<div>
+			<AdminTaskbar />
+			<Box component="main" sx={{ flexGrow: 1, p: 15 }}>
+				<Toolbar />
+				<Typography paragraph align="center">
+				Message Center
+				</Typography>
+			</Box>
+		</div>
+		
```

Comment:
**suggestion (non-blocking):** In teams we usually make sure that our prettier/eslint works for everyone so we avoid formatting inconsistencies, like empty lines, trailing spaces, indentation. We discussed this in #28 as well